README.md

```diff
@@ -76,9 +76,35 @@ The major components of EAA are:
 
 Please refer to [Inclavare Containers Roadmap](ROADMAP.md) for the details. This document outlines the development roadmap for the Inclavare Containers project.
 
+# Usage
+
+To give Inclavare Containers a quick try, please use the Inclavare Containers development [docker images](https://hub.docker.com/repository/docker/inclavarecontainers/dev) by following the steps below:
+
+Step 1-2 are to be done on the host OS (Linux):
+
+1. Please follow [Intel SGX Installation Guide](https://download.01.org/intel-sgx/sgx-linux/2.13/docs/Intel_SGX_Installation_Guide_Linux_2.13_Open_Source.pdf) to install Intel SGX driver, Intel SGX SDK & PSW for Linux.
+
+2. Run the Inclavare Containers Docker container.
+
+> {ubuntu18.04, centos8.2, alinux2} systems are supported.
+
+- For OOT driver:
+
+```shell
+docker run -it --device /dev/isgx -v /var/run/aesmd:/var/run/aesmd \
+  inclavarecontainers/dev:[version]-[system]
+```
+
+- For DCAP driver:
+
+```shell
+docker run -it -v /dev/sgx_enclave:/dev/sgx/enclave -v /dev/sgx_provision:/dev/sgx/provision \
+  -v /var/run/aesmd:/var/run/aesmd inclavarecontainers/dev:[version]-[system]
+```
```

Comment:
Please move the above changes to the Building section.

And use the following content instead:

```
# Building

It's recommended to use [Inclavare Containers development docker image](...) to build Inclavare Containers from scratch.

Note that the environment of launching Inclavare Containers development docker image must be capable of hardware TEE and install the corresponding software stack, e.g, Intel SGX and [Intel SGX SDK & PSW for Linux](https://download.01.org/intel-sgx/latest/linux-latest/docs/Intel_SGX_Installation_Guide_Linux_2.13.3_Open_Source.pdf).

The exact command to run the docker image requires to be specified explicitly according to the type of SGX device driver.

For legacy out-of-tree driver:

docker run -it -v /var/run/aesmd:/var/run/aesmd \
  -device /dev/isgx \
  inclavarecontainers/dev:$version-$os

For DCAP and in-tree driver:

docker run -it -v /var/run/aesmd:/var/run/aesmd \
  -v /dev/sgx_enclave:/dev/sgx/enclave -v /dev/sgx_provision:/dev/sgx/provision \
  inclavarecontainers/dev:$version-$os

where:
- $version denotes the version of Inclavare Containers in use.
- $os denotes the OS type of development docker image, which may be ubuntu18.04, centos8.2 or alinux2.
```